CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## 1.1.3 2017-08-09
+
+The 1.1.3 release of Linkerd is mostly focused on improving our HTTP/2 support,
+including better support for gRPC.  Linkerd now supports automatic retries in
+HTTP/2 for retryable requests, even when there are (small) request or response
+bodies.
```

Comment:
The last part of this sentence isn't really clear to me. Does this mean that automatic retries only happen for requests or responses of a certain size? What's the size threshold after which a request is not retried?